examples/logger.py

```diff
@@ -3,20 +3,51 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
-
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
-
```

Comment:
You should keep these blank lines.
Please read [PEP8](https://peps.python.org/pep-0008/#imports).
Also, I improved the developer guide #2.